src/agents/user_agent.py

```diff
@@ -1,33 +1,38 @@
+import calendar
 import json
+import time
+from dataclasses import dataclass
 from typing import List
+
 from spade.agent import Agent
 from spade.behaviour import CyclicBehaviour
 from spade.message import Message
 from spade.template import Template
 
-from dataclasses import dataclass
-import calendar
-import time
-
 from constants import DEFAULT_HOST, MESSAGE_TIMEOUT
+from logger import logger
 from messages.check_offers import CheckOffers
-from messages.consolidated_offers import ConsolidatedOffers
+from messages.consolidated_offers import ConsolidatedOffers, Offer
 from messages.modify_reservation import ModifyReservation
 from messages.reservation_request import RequestReservation
 from messages.response import ReservationResponse
 
 
 class User(Agent):
-    def __init__(self, jid, password):
+    def __init__(self, jid, password, coordinators_jids=[]):
         super().__init__(jid, password)
-        self.coordinators_jids = None
+        self.coordinators_jids = coordinators_jids
         self.pending_reservation = None
         # dictionary of active reservations {reservation_id: Reservation}
         self.active_reservations = {}
 
     def _validate_timeslot(self, time_start, time_stop):
-        current_time = calendar.timegm(time.gmtime())  # Get current timestamp in UTC timezone
-        if not (time_start < time_stop and time_start > current_time):
+        # current_time = calendar.timegm(time.gmtime())  # Get current timestamp in UTC timezone
+        # if not (time_start < time_stop and time_start > current_time):
+        #     raise ValueError("Invalid timeslot")
```

Comment:
Please remove commented code if not needed

Reply:
I would prefer to improve the time validation a bit and created a TODO comment with that code in the other PR. And this code could be useful.